### PR TITLE
format + wrong title + GUI-Core element

### PR DESCRIPTION
**Pull request by jrobinAV**

### CODE REVIEW

File: docs/knowledge_base/tutorials/complete_application/step_01/step_01.md [hunk 1]
```diff
@@ -56,19 +60,23 @@ The Data Visualization page includes the following visual elements:
...
+Taipy maintains a distinct state for every client connection. This state stores the values of 
+all variables used in the user interface. For example, modifying *n_week* through a slider will 
 update *state.n_week*, not the global Python variable *n_week*. Each client has its own state, 
 ensuring that changes made by one client don't affect others.
 
-## [Callbacks](../../../../manuals/gui/callbacks.md)
+## Callbacks
 
-You can include callbacks in each visual element, enabling you to modify variables according to user actions. For further details, explore local callbacks and global callbacks.
+You can include [Callbacks](../../../../manuals/gui/callbacks.md) in each visual element,
```
Comment (FabienLelaquais) [inline docs/knowledge_base/tutorials/complete_application/step_01/step_01.md]: "You can include \[Callbacks\](../../../../manuals/gui/callbacks.md) in each visual element" ->
"Most visual element include  \[callbacks\](../../../../manuals/gui/callbacks.md)"

File: docs/knowledge_base/tutorials/complete_application/step_01/step_01.md [hunk 2]
```diff
@@ -56,19 +60,23 @@ The Data Visualization page includes the following visual elements:
...
+You can include [Callbacks](../../../../manuals/gui/callbacks.md) in each visual element, 
+enabling you to modify variables according to user actions. For further details, explore local 
+callbacks and global callbacks.
 
 - *state*: The state object containing all the variables.
 - The name of the modified variable. (optional)
 - Its new value. (optional)
 
-Here's an example of `on_change()` function to update *state.dataset_week* based on the selected week from the slider:
+Here's an example of `on_change()` function to update *state.dataset_week* based on the selected
```
Comment (FabienLelaquais) [inline docs/knowledge_base/tutorials/complete_application/step_01/step_01.md]: "of \`on_change()\` function" -> "of setting the \`on_change\` callback"

File: docs/knowledge_base/tutorials/cycles_scopes/index.md [hunk 3]
```diff
@@ -242,9 +242,9 @@ Scenario 2: Primary? False True
 - `tp.get_primary(<Cycle>)`: returns the primary scenario of the Cycle.
 
 
-### GUI-Core visual elements
+### Scenario management visual elements
 
-You can utilize GUI-Core elements to control Cycles. Cycles can be seen in either the
+You can use Scenario management visual elements to control Cycles. Cycles can be seen in either the
 *scenario_selector* or *data_node_selector*. Additionally, it's possible to designate a scenario
 as primary directly through the `scenario` visual element.
```
Comment (FabienLelaquais) [inline docs/knowledge_base/tutorials/cycles_scopes/index.md:249]: Link to the `scenario` control?
"Scenario management visual elements" -> "Scenario management controls" (+ link to .../gui/viselements/controls/#scenario-management-controls)?

File: docs/knowledge_base/tutorials/cycles_scopes/index.md [hunk 4]
```diff
@@ -242,9 +242,9 @@ Scenario 2: Primary? False True
 - `tp.get_primary(<Cycle>)`: returns the primary scenario of the Cycle.
 
 
-### GUI-Core visual elements
+### Scenario management visual elements
 
-You can utilize GUI-Core elements to control Cycles. Cycles can be seen in either the
+You can use Scenario management visual elements to control Cycles. Cycles can be seen in either the
 *scenario_selector* or *data_node_selector*. Additionally, it's possible to designate a scenario
```
Comment (FabienLelaquais) [inline docs/knowledge_base/tutorials/cycles_scopes/index.md]: \*scenario_selector\* -> \`scenario_selector\`
\*data_node_selector\* -> \`data_node_selector\`

**Review by FabienLelaquais — APPROVED** [review]
Minor, mainly typos

File: docs/knowledge_base/tutorials/complete_application/step_01/step_01.md (hunk 1 above)
Comment (jrobinAV) [inline docs/knowledge_base/tutorials/complete_application/step_01/step_01.md]:
```suggestion
Most visual element include [callbacks](../../../../manuals/gui/callbacks.md), 
```

File: docs/knowledge_base/tutorials/complete_application/step_01/step_01.md (hunk 2 above)
Comment (jrobinAV) [inline docs/knowledge_base/tutorials/complete_application/step_01/step_01.md]:
```suggestion
Here's an example of of setting the `on_change` callback function to update *state.dataset_week* based on the selected 
```

File: docs/knowledge_base/tutorials/cycles_scopes/index.md (hunk 4 above)
Comment (jrobinAV) [inline docs/knowledge_base/tutorials/cycles_scopes/index.md]:
```suggestion
 `scenario_selector` or `data_node_selector`. Additionally, it's possible to designate a scenario
```

**Pull request merged**